test/uploads.js

```diff
@@ -6,37 +6,36 @@
 var should = require("should")
     , strava = require("../");
 
-describe.skip('uploads_test', function(){
-
+describe('uploads_test', function(){
     describe('#post()', function() {
-
-        it('should post example activity to strava', function(done) {
-
-            strava.uploads.post({
-
-                activity_type:'run'
-                , data_type:'gpx'
-                , name:'test activity'
-                , file:'data/Evening_Run.gpx'
-                , statusCallback: function(err,payload) {
-
-                    //console.log(payload)
-                    done();
-                }
-
-            },function(err,payload){
-
-                if(!err) {
-                    //console.log(payload);
-                    //(payload.resource_state).should.be.exactly(3);
-                    //(payload.city).should.be.exactly("Seattle");
-                }
-                else {
-                    console.log(err);
-                }
-
-                //done();
-            });
+        it('should upload a GPX file', function(done) {
+            new Promise((resolve, reject) => {
```

Comment:
This breaks the tests on node `v0.12.x` due to the use of arrow functions! We'd have to add babel support to the repo to support these changes in a non breaking way

```
Running "simplemocha:all" (simplemocha) task
Warning: /var/www/strava/test/uploads.js:12
            new Promise((resolve, reject) => {
                                          ^^
Unexpected token => Used --force, continuing.
```